src/tactic/congr.lean

```diff
@@ -0,0 +1,184 @@
+/-
+Copyright (c) 2020 Floris van Doorn. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Floris van Doorn
+-/
+import tactic.lint
+import tactic.ext
+
+/-!
+# Congruence and related tactics
+
+This file contains the tactic `congr'`, which is an extension of `congr`, and various tactics
```

Comment:
Could you add a sentence here about the difference between `congr` and `congr'`?